spec/jobs/metricsforwarder/metricsforwarder_spec.rb

```diff
@@ -44,42 +49,56 @@
 
   context 'config/metricsforwarder.yml' do
 
```

Comment:
we seem to have lost the `does not set username nor password if not configured` test